ibm_catalog.json

```diff
@@ -64,6 +64,18 @@
               {
                 "title": "Quickstart variation",
                 "description": "Configured to use an Event Streams instance with either the lite or standard plan, including Kafka topics and optional add-on integrations, editable to fit your use case.."
+              },
+              {
+                "title": "Sets up logging for the Event Streams instance",
+                "description": "Optionally, you can deploy Cloud automation for Cloud Logs to route, alert, and visualize platform logs that are generated by your Event Streams instance. [Learn more](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-streaming_data_cloud_logs)"
+              },
+              {
+                "title": "Sets up monitoring operational metrics for the Event Streams instance",
+                "description": "Optionally, you can deploy Cloud automation for Cloud Monitoring to measure how users and applications interact with your Event Streams instance. [Learn more](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-metrics)"
+              },
+              {
+                "title": "Sets up activity tracking for the Event Streams instance",
+                "description": "Optionally, you can deploy Cloud automation for Activity Tracker Event Routing to route and securely store auditing events that are related to your Event Streams instance. [Learn more](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-at_events)"
```

Comment:
These are in the wrong section - They should be added as offering level features, not variation level

Reply:
You should probably also have an offering level features for SCC-WP. And perhaps point the Learn more link to https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-manage-security-compliance

Reply:
@whoffler This is something you will need to close on. Suggest to propose some text - and if needed we can run it by content. Account infra is used everywhere, so lets get closure on it now so we can use everywhere. Suggest to add the final approved text to the GIT issue epic